gcloud/resource_manager/client.py

```diff
@@ -49,7 +49,7 @@ class Client(BaseClient):
     _connection_class = Connection
 
     def new_project(self, project_id, name=None, labels=None):
-        """Creates a :class:`.Project` bound to the current client.
+        """Create a :class:`.Project` bound to the current client.
```

Comment:
> english language usage

Out of my depth there. Seems a real challenge to get lots of contributors on the same page with that (without someone dedicated to docs).